dpctl/__init__.pxd

```diff
@@ -29,4 +29,4 @@
 
 from dpctl._sycl_core cimport *
 from dpctl._memory import *
-
+from dparray import *
```

Comment:
As I know it is an absolute import. How it is possible to import dparray which is inside dpctl package via absolute path?
```suggestion
from .dparray import *
```